product_docs/docs/pgd/5/ddl/ddl-replication-options.mdx

```diff
@@ -40,6 +41,4 @@ of executed DDL commands. You must then run the DDL manually on all nodes.
     cause the whole PGD group to stop replicating if conflicting DDL or DML
     executes concurrently.
 
-The `bdr.ddl_replication` parameter can be set only by the bdr_superuser,
-by superuser, or in the `postgres.conf` configuration file.
-
+Only only the bdr_superuser or superuser can set the `bdr.ddl_replication` parameter can be set. Or, you can set it in the `postgres.conf` configuration file.
```

Comment:
```suggestion
Only the bdr_superuser or superuser can set the `bdr.ddl_replication` parameter. It can also be set in the `postgres.conf` configuration file.
```